docs/DeveloperGuide.md

```diff
@@ -196,80 +200,90 @@ Guarantees: Editing of internship is successful
   1b1. InternHunter displays an error message and informs the user that the index is out of bounds. <br/>
   Use case resumes from step 1.
 
-### Use case: UC08 - Add an internship application
+### Use case: UC08 - Add an application
 
 #### MSS
 
-Guarantees: Adding of internship application is successful
+Precondition: User is on the Companies page and already has an existing list of internships of a company <br/>
+Guarantees: Adding of application is successful
 
-1.  User requests to add an internship application and provides the relevant details.
-2.  InternHunter adds the internship application to the list of internship applications.
+1.  User requests to apply for an internship and provides the index and relevant details.
+2.  InternHunter adds the application to the list of applications and prompts the user to switch to the Companies tab
```

Comment:
Is it switching to the applications tab?

Reply:
Yes, thanks for noticing.